core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -1193,6 +1193,24 @@ private void checkColumnUniquenessForFilterWithConstantColumns(String sql) {
         .assertThatUniqueKeysAre(bitSetOf());
   }
 
+  @Test void testColumnUniquenessForLeftJoinOnLimit1() {
```

Comment:
it is customary to add a JavaDoc comment to the test indicating the JIRA issue that is being addressed. This makes it easier for maintainers to understand the rationale for a test. You can find lots of examples in the code about how that is supposed to be structured, please follow the pattern.

Reply:
Added the comment. Thanks for your reminder